README.md

```diff
@@ -54,7 +54,7 @@ $ oc apply -f deploy/crd/operatorsource.crd.yaml
 $ oc apply -f deploy/service_account.yaml
 $ oc apply -f deploy/role.yaml
 $ oc apply -f deploy/role_binding.yaml
-$ oc apply -f deploy/marketplace.csv.yaml
+$ oc apply -f deploy/marketplace.v0.0.1.clusterserviceversion.yaml
 ```
 
 ### Deploying the Marketplace Operator with Kubernetes
```

Comment:
There are a couple of references just above this change that use `oc project openshift-marketplace` that need to be updated to `oc project openshift-operators`

Reply:
Good catch. @awgreene please address that in this commit.
/hold

Reply:
I have addressed this comment in the latest commit.